content/download/weekly/index.adoc

```diff
@@ -0,0 +1,50 @@
+---
+layout: simplepage
+title: Weekly Release Line
+---
+:toc:
+
+The Jenkins _Weekly Release Line_ delivers bug fixes and new features rapidly to users and plugin developers who need them.
+These releases are generally delivered on a weekly cadence, hence the name.
+For more conservative users, there is also a 
+link:/download/lts/[Long-Term Support (LTS)] line.
+
+## Model
+
+The "weekly" release cadence is **aspirational**.
+Generally we target releasing Jenkins every week, but sometimes we do out-of-order releases.
+To get notifications about new releases, it is possible to subscribe to link:https://www.jenkins.io/changelog/rss.xml[releases RSS feed].
+
+Conditions for new releases of the _Weekly Release Line_:
+
+* **Regular releases** with all features, improvements and bug fixes integrated since the last release.
+** Currently these releases happen on a weekly basis on Tuesdays.
```

Comment:
As a personal preference would make this a dl so we don't need nested list items, that seems excessive.

See e.g. https://www.jenkins.io/security/for-maintainers/#support-for-maintainers or https://www.jenkins.io/security/#learn-more for examples.

Reply:
I think the list needs a bit more entries to get seriously impacted by look&feel. If we extend the documentation in this section any more, I commit to update the layout